vm-virtio/src/vhost_user/handler.rs

```diff
@@ -79,8 +79,10 @@ impl<S: VhostUserMasterReqHandler> VhostUserEpollHandler<S> {
         )
         .map_err(Error::EpollCtl)?;
 
```

Comment:
Typo in the commit message subject.

Reply:
And, honestly, I can't follow the body either.

Reply:
Ok, I will update it.